snippets/language-ruby.cson

```diff
@@ -114,6 +114,9 @@
   'def … end':
     'prefix': 'def'
     'body': 'def ${1:method_name}\n\t$0\nend'
+  'initialize':
+    'prefix': 'init'
+    'body': 'def initialize${1:(${2:arguments, …})}\n\t$0\nend'
```

Comment:
@izuzak I just thought `arguments, ...` would suggest/encourage user to define `initialize` with multiple args separated by commas (if necessary that is).

When you `init <tab> <tab> foo <tab> puts foo` you will still end up with:

```ruby
def initialize(foo)
  puts foo
end
```

... or if you `init <tab> <enter> puts 'hello'` this is what you will going to get:

```ruby
def initialize
  puts 'hello'
end
```

No strong feelings about this though. I like my version (nested placeholders + more verbose suggestions) quite a lot, but still I am okay with whatever.